eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/configuration/ForceImportDeciderConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.dbsnp.configuration;
+
+import org.springframework.batch.core.job.flow.JobExecutionDecider;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import uk.ac.ebi.eva.accession.dbsnp.deciders.ForceImportDecider;
+
+import static uk.ac.ebi.eva.accession.dbsnp.configuration.BeanNames.FORCE_IMPORT_DECIDER;
+
+@Configuration
+public class ForceImportDeciderConfiguration {
```

Comment:
Given how simple this class is, maybe the `@Bean` method could be moved to the flow configuration. But it is just a matter of style, nothing really wrong with this approach.